release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2686,6 +2686,8 @@ A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
 
 * Currently, the *YAML* tab of some pages in the web console stops unexpectedly on some browsers when the multicluster engine for Kubernetes operator (MCE) is installed. The following message is displayed: "Oh no! Something went wrong." (link:https://issues.redhat.com/browse/OCPBUGS-29812[*OCPBUGS-29812*])
 
+* If you have IPsec enabled on your cluster, you must disable it prior to upgrading to {product-title} 4.15. There is a known issue where pod-to-pod communication might be interupted or lost when updating to 4.15 without disabling IPsec. For information on disabling IPsec, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption]. (link:tobug)
```

Comment:
add link to the known issue?

Reply:
Same comment as here: https://github.com/openshift/openshift-docs/pull/83462#discussion_r1801142548